esignet-default.properties

```diff
@@ -134,7 +161,7 @@ mosip.esignet.authenticator.ida.otp-channels=email,phone
 mosip.esignet.ida.vci-user-info-cache=userinfo
 mosip.esignet.ida.vci-exchange-id=mosip.identity.vciexchange
 mosip.esignet.ida.vci-exchange-version=1.0
-mosip.esignet.ida.vci-exchange-url=${mosip.ida.auth.url}/idauthentication/v1/vci-exchange/delegated/${mosip.esignet.misp.license.key}/
+mosip.esignet.ida.vci-exchange-url=https://${mosip.api.internal.host}/idauthentication/v1/vci-exchange/delegated/${mosip.esignet.misp.license.key}/
```

Comment:
change this  to previous value ${mosip.ida.auth.url}

Reply:
done